block/src/Pos/Block/Network/Retrieval.hs

```diff
@@ -343,11 +343,13 @@ getProcessBlocks enqueue nodeId lcaChild newestHash = do
                 logWarning msg
                 throwM $ DialogUnexpected msg
             Right blocks -> do
+                let hashes = getOldestFirst $ map (headerHash . view blockHeader) blocks
```

Comment:
Btw, `view blockHeader` is most likely redundant here, because `headerHash` is very polymorphic.